executor/distsql.go

```diff
@@ -111,6 +113,17 @@ func (task *lookupTableTask) Swap(i, j int) {
 	task.rows[i], task.rows[j] = task.rows[j], task.rows[i]
 }
 
+func (task *lookupTableTask) buildResultChunk() {
+	if task.numRows <= 0 {
+		return
+	}
+	if task.rows[0].Chunk() != nil {
+		task.chk = chunk.Renew(task.rows[0].Chunk(), task.rows[0].Chunk().NumRows())
```

Comment:
renew a `Chunk` here will take up extra memory space, which may lead to worse performance.